pkg/api/types.go

```diff
@@ -5,6 +5,10 @@ import docker "github.com/fsouza/go-dockerclient"
 const (
 	DefaultNamespace    = "io.openshift.s2i."
 	KubernetesNamespace = "io.k8s."
+	DefaultIgnoreFile   = ".s2iignore"
+	DefaultUploadSubdir = "upload"
+	DefaultSrcSubdir = "src"
```

Comment:
Was unaware of the constants in scripts.go.  Thanks for the pointer, will be part of next push.

Quick follow up:  the use of "/" in those constants in scripts.go - are we concerned about running sti on windows?  Using filepath.Join is suppose to take care of "/" vs. "\"